doc/source/configuration/wazuh.rst

```diff
@@ -317,16 +319,45 @@ You will need to create two files matching the following pattern:
 - ``{{ dashboard_node_name }}-key.pem`` for the private key
 - ``{{ dashboard_node_name }}.pem`` for the certificate
 
-Drop these files into ``$KAYOBE_CONFIG_PATH/environments/<environment>/wazuh/wazuh-certificates/`` if
-using the kayobe environments feature, or ``$KAYOBE_CONFIG_PATH/wazuh/wazuh-certificates/`` if not.
-The key for the external certificate should be in PKCS#8 format (in its header it may have BEGIN
+In order to use externally generated certificates, you must first deploy Wazuh Ansible as normal in order to generate the base certificates. 
+This is because Wazuh will require to deploy with some sort of certificates, as well as, not every Wazuh service needing a custom external certificate; 
+for example the ``root`` and ``admin`` certificates. Therefore the ``<...>/wazuh/wazuh-certificates`` directory must be created and populated by ``wazuh-cert-tool.sh``, which it won't do if the ``<...>/wazuh/wazuh-certificates`` directory already exists.
```

Comment:
While the existing text is correct, the "why" section makes it much more confusing. 

```suggestion
To use externally generated certificates, you must first deploy Wazuh Ansible as normal to generate the base certificates. The custom certificates can then be added to this bundle.
```